vnext/Shared/RuntimeOptions.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+
+namespace Microsoft::React {
+
+/// <summary>
+/// Sets a global boolean value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global key</param>
+void __cdecl SetRuntimeOptionBool(std::string &&name, bool value) noexcept;
+
+/// <summary>
+/// Sets a global signed integer value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global boolean key</param>
+void __cdecl SetRuntimeOptionInt(std::string &&name, std::int32_t value) noexcept;
+
+/// <summary>
+/// Retrieves a global boolean value for the given key.
+/// </summary>
+/// <param name="name">Global boolean key</param>
+/// <returns>Value stored for the given key, or false if the entry doesn't exist (default)</returns>
+const bool __cdecl GetRuntimeOptionBool(const std::string &name) noexcept;
```

Comment:
>const std::string &name [](start = 40, length = 23)

Is a std::string needed/advantageous in the function implementation? If not, wouldn't a 'const char*' suffice, and potentially prevent a std::string allocation at the call sites?

Reply:
Good point. This API would likely just use a string literal as argument, and not a variable.
We may change this to `const char*`, or `std::string_view`.